src/njoy21/input/VIEWR/Card1/test/Card1.test.cpp

```diff
@@ -0,0 +1,31 @@
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "Parsing valid VIEWR Card1 input" ){
+  { 
+    iRecordStream<char> iss( std::istringstream( "21 22" ) );
+    VIEWR::Card1 card1( iss );
+    REQUIRE( card1.infile.value == 21 );
+    REQUIRE( card1.nps.value == 22 );
+  }
+  {
+    iRecordStream<char> iss( std::istringstream( "21 -22") );
```

Comment:
I'm a little surprised that mixed mode (i.e., positive and negative values) can be used here; though, I can't seem to find anything in the documentation that disallows this.